src/span-data.js

```diff
@@ -72,7 +76,8 @@ function SpanData(agent, trace, name, parentSpanId, isRoot, skipFrames) {
         callSite.getColumnNumber()));
     });
     this.span.setLabel(TraceLabels.STACK_TRACE_DETAILS_KEY,
-      JSON.stringify({stack_frame: stackFrames}));
+      traceUtil.truncate(JSON.stringify({stack_frame: stackFrames}),
```

Comment:
nit: add a comment that we are intentionally not respective `config.maxLabelValueSize` here.